### ISSUE TITLE
How to display the predicted result graph?

### ISSUE THREAD

**Issue by zengyuncheng** [VeritasYin/STGCN_IJCAI-18#30]
Thanks a lot for sharing!
I've got four files to output: checkpoint/data-00000-of-00001/index/meta.
But I cannot get the predictive graph to show up.
Could you please tell me how to display the predicted result graph?

**Comment by ahariri13** [comment]
@zengyuncheng how did you manage to display it ? Thanks in advance

**Comment by VeritasYin** [comment]
Apologized for the confusion. In our released code, we did not provide a function or API for ploting the final prediction. But you can use Tensorboard for visulizing the training loss, the copy loss and the change of learning rate from those log files.
